src/components/shared/CustomModalWindow/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { FC } from 'react';
+import { createTheme, Dialog, DialogContent, DialogTitle, IconButton, ThemeProvider } from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+import { useDispatch, useSelector } from 'react-redux';
+import { CustomInput } from '../CustomInput';
+import { CustomButtonComponent } from '../CustomButton';
+import UploadBackgroundLeft from '../../../assets/img/modal-background.svg';
+import UploadBackgroundRight from '../../../assets/img/modal-background2.svg';
+import { closeModalWindow } from '../../../store/board-service/actions';
+import { getModalIsOpen } from '../../../store/board-service/selectors';
+
+interface CustomModalProps {
+	title: string;
+	onSubmit: () => undefined;
+	onClick: () => undefined;
+}
+
+export const CustomModal: FC<CustomModalProps> = ({ title, onSubmit, onClick }) => {
+	const isModalOpen: boolean = useSelector(getModalIsOpen);
+	const dispatch = useDispatch();
+
+	const theme = createTheme({
+		components: {
+			MuiDialogTitle: {
+				styleOverrides: {
+					root: {
+						fontWeight: '600',
+						fontSize: '22px',
+						lineHeight: '27px',
+						color: '#030327',
+						marginBottom: '20px',
+						padding: '0px 0px',
+					},
+				},
+			},
+			MuiDialogContent: {
+				styleOverrides: {
+					root: {
+						display: 'flex',
+						flexDirection: 'column',
+						justifyContent: 'center',
+						alignItems: 'center',
+						width: '514px',
+						height: '300px',
+						border: '1px solid #dddddd',
+						borderRadius: '10px',
+						padding: '15px',
+						background: `url(${UploadBackgroundRight}) 90% 100% no-repeat, url(${UploadBackgroundLeft}) center/cover no-repeat, #f6f6f6`,
+					},
+				},
+			},
+			MuiPaper: {
+				styleOverrides: {
+					root: {
+						borderRadius: '10px',
+					},
+				},
+			},
+			MuiDialog: {
+				styleOverrides: {
+					container: {
+						position: 'absolute',
+						top: '5%',
+						left: '30%',
+						height: '380px',
+					},
+				},
+			},
+		},
+	});
+
+	return (
+		<ThemeProvider theme={theme}>
```

Comment:
your modal is not shared now. if I wish to show a modal with some text and two buttons, can I use that?